packages/@okta/vuepress-site/docs/concepts/manage-user-creds/index.md

```diff
@@ -0,0 +1,190 @@
+
+Okta supports two types of authorization servers:
+
+* The [org authorization server](/docs/concepts/auth-servers/#org-authorization-server) is built in and supports simple SSO for OIDC apps or for obtaining access tokens for Okta APIs. It doesn't support custom scopes, claims, or policies.
+* A [custom authorization server](/docs/concepts/auth-servers/#custom-authorization-server) allows you to create and apply authorization policies to secure your own APIs. You can also define custom scopes and claims, and integrate with API gateways. It's recommended to assign one authorization server per API product to maintain separate policies and token expiration times, and to prevent scope name collisions.
+
+## Why secure token management is critical
+
+Securely managing your tokens is essential to mitigate risks that directly threaten the protection of user data, such as token theft and account takeover.
+
+Risk areas include:
+
+* Compromised user identity and data: If unauthorized parties access tokens, a user's identity could be compromised. Without proper management, token leakage can pose a significant risk to user data. However, OAuth 2.0 tokens used for authorizing API requests are short-lived. You can configure them to expire after a specific duration, which reduces the window of opportunity for attackers to misuse them. In contrast, using passwords directly can remain valid indefinitely unless explicitly changed.
+* Vulnerabilities in web environments: Web browsers introduce unique security challenges, particularly from Cross-Site Scripting (XSS) and Cross-Site Request Forgery (CSRF) vulnerabilities.
+  * XSS attacks involve malicious scripts running inside your app. These scripts can execute any potential actions that your app can perform. Frequently, attackers use XSS to steal authentication tokens stored in the browser.
+  * CSRF allows attackers to trick an authenticated browser into making unintended requests.
+* Concurrency issues: Managing tokens involves storage, retrieval, and refresh operations. In complex apps, especially during initialization or with concurrent operations, multiple calls to token storage can lead to shared resource contention.
+* Lack of granular control and scope restrictions: If tokens aren't scoped appropriately, any security compromise can lead to broader unauthorized access to resources, increasing the potential damage. Defining granular and specific scopes within authorization servers is a best practice.
+* Insecure token transmission: All interactions and redirects involving users, apps, and Okta must be secured through HTTPS (SSL/TLS) connections to protect credentials and tokens from interception. Pass access tokens exclusively through an HTTP Authorization header. Also, never encode tokens into a payload or URL, as these could be logged or cached.
+<!-- [For later: The Okta Client SDK's token management system is designed to mitigate these issues and ensure app stability.] -->
+
+### Token lifecycle
```

Comment:
This section seems dropped in .... it's confusing in the flow why'd we jump to this. Perhaps needs an introductory paragraph to understand the mgmt aspects behind the lifecycle